### PR TITLE
Replace al 0s in Mirte hostname with As at generate time

### LINKED ISSUE #28: Remove 0s from Mirte-XXXXXX

### PR DESCRIPTION
**Pull request by ArendJan**
Fix #28 by replacing all 0s with As when generating a new hostname

### CODE REVIEW

File: network_setup.sh [hunk 1]
```diff
@@ -111,6 +111,8 @@ $MIRTE_SRC_DIR/mirte-install-scripts/usb_ethernet.sh
 if [ ! -f /etc/ssid ] || [[ $(cat /etc/hostname) == "Mirte-XXXXXX" ]]; then
 	UNIQUE_ID=$(openssl rand -hex 3)
 	MIRTE_SSID=Mirte-$(echo ${UNIQUE_ID^^})
+	# replace al 0s with As to make it clearer for users when looking at the leds, as 0 is difficult to recognize
+	MIRTE_SSID=${MIRTE_SSID//0/A}
```
Comment (mklomp) [inline network_setup.sh:115]: I would not replace 0's with A's, but leave out the 0's from the RNG. I do not think there is an easy way of doing this with openssl. So I would replace:

UNIQUE_ID=$(tr -cd "1-9A-F" < /dev/urandom | head -c 6)

to make sure 0's are not in there.

**Review by mklomp — COMMENTED** [review]
Check my comment. Also, we need to test if this does indeed generate unique IDs for multiple SD's. And if we need an v0.1-rc3, this might also be at v0.1.

**Comment by mklomp** [comment]
O, and..... at soem point we might also generate this based on the MAC-address rather then unique for each SD.

File: network_setup.sh (hunk 1 above)
Comment (ArendJan) [reply network_setup.sh]: I've changed it and I'm currently generating an SD to test that they differ

**Comment by ArendJan** [comment]
Just tested it with the same image, flashed it onto the same SD, same setup, and the IDs were different: Mirte-1E9529 and Mirte-833F49 and Mirte-B95DF1.

**Pull request merged**